WooCommerce/Classes/Styles/UIColor+MurielColors.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+extension UIColor {
+    /// Get a UIColor from the Muriel color palette
+    ///
+    /// - Parameters:
+    ///   - color: an instance of a MurielColor
+    /// - Returns: UIColor. Red in cases of error
+    class func muriel(color murielColor: MurielColor) -> UIColor {
+        let assetName = murielColor.assetName()
+        let color: UIColor?
+
+        // This is temporary work around as there's a bug in the
+        // GM seed of Xcode 11 which causes loading colors from asset
+        // catalogs to fail (54325712)
```

Comment:
The [Apple bug (54325712)](https://developer.apple.com/documentation/xcode_release_notes/xcode_11_release_notes) has been resolved. This workaround can be removed. (I'm sorry, I don't know how much work this will cause you nor the full scope of the code that will need removed.)

Reply:
You can see the WPiOS PR to remove this workaround here: https://github.com/wordpress-mobile/WordPress-iOS/pull/12931/files